### PR TITLE
Fix "Add new vault" navigation + Fix onboarding logic

### PR DESCRIPTION
**Pull request by rcoderdev**

### CODE REVIEW

File: frontend/src/vault/add/AddVaultPage.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  const { t } = useTranslation();
+
+  return (
+    <VStack fill>
+      <PageHeader primaryControls={<PageHeaderBackButton />} />
+      <PageContent gap={16}>
+        <ProductLogoBlock />
+        <VStack gap={16}>
+          <Link to={setupVaultPath}>
+            <Button as="div">{t('create_new_vault')}</Button>
```
Comment (antoni0dev) [inline frontend/src/vault/add/AddVaultPage.tsx:21]: what's the reasoning behind setting a button as a div? if clickable, it gotta be a button

Comment (antoni0dev) [inline frontend/src/vault/add/AddVaultPage.tsx:21]: semantic conflict - I think we should use either <a /> or <button> not nesting them. Maybe we could use <Button as="link" />

File: frontend/src/vault/add/AddVaultPage.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          </Link>
+          <Link to={importVaultPath}>
+            <Button as="div" kind="outlined">
```
Comment (antoni0dev) [inline frontend/src/vault/add/AddVaultPage.tsx:24]: semantic conflict

**Review by antoni0dev — APPROVED** [review]
negligible comments :)

File: frontend/src/vault/add/AddVaultPage.tsx (hunk 1 above)
Comment (rcoderdev) [reply frontend/src/vault/add/AddVaultPage.tsx:21]: In HTML it's not allowed to have a `<button>` inside of `<a>`, that's why we render this button as a `div`

**Pull request merged**